src/OpenTelemetry.Api/.publicApi/net452/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,327 @@
+OpenTelemetry.Context.Propagation.BaggagePropagator.Fields.get -> System.Collections.Generic.ISet<string>
+OpenTelemetry.Context.Propagation.BaggagePropagator.Inject<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Action<T, string, string> setter) -> void
+OpenTelemetry.Context.Propagation.CompositePropagator
+OpenTelemetry.Context.Propagation.CompositePropagator.CompositePropagator(System.Collections.Generic.IEnumerable<OpenTelemetry.Context.Propagation.IPropagator> propagators) -> void
+OpenTelemetry.Context.Propagation.CompositePropagator.Extract<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Func<T, string, System.Collections.Generic.IEnumerable<string>> getter) -> OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.CompositePropagator.Fields.get -> System.Collections.Generic.ISet<string>
+OpenTelemetry.Context.Propagation.CompositePropagator.Inject<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Action<T, string, string> setter) -> void
+OpenTelemetry.Context.Propagation.IPropagator
+OpenTelemetry.Context.Propagation.IPropagator.Extract<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Func<T, string, System.Collections.Generic.IEnumerable<string>> getter) -> OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.IPropagator.Fields.get -> System.Collections.Generic.ISet<string>
+OpenTelemetry.Context.Propagation.IPropagator.Inject<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Action<T, string, string> setter) -> void
+OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.PropagationContext.ActivityContext.get -> System.Diagnostics.ActivityContext
+OpenTelemetry.Context.Propagation.PropagationContext.Baggage.get -> OpenTelemetry.Baggage
+OpenTelemetry.Context.Propagation.PropagationContext.Equals(OpenTelemetry.Context.Propagation.PropagationContext value) -> bool
+OpenTelemetry.Context.Propagation.PropagationContext.PropagationContext(System.Diagnostics.ActivityContext activityContext, OpenTelemetry.Baggage baggage) -> void
+OpenTelemetry.Context.Propagation.TextMapPropagator
+OpenTelemetry.Context.Propagation.TextMapPropagator.Extract<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Func<T, string, System.Collections.Generic.IEnumerable<string>> getter) -> OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.TextMapPropagator.Fields.get -> System.Collections.Generic.ISet<string>
+OpenTelemetry.Context.Propagation.TextMapPropagator.Inject<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Action<T, string, string> setter) -> void
+OpenTelemetry.Context.Propagation.TextMapPropagator.TextMapPropagator() -> void
+OpenTelemetry.Context.RemotingRuntimeContextSlot<T>
```

Comment:
does this need to be public? Sounds like an implementation detail

Reply:
can you comment here?

Reply:
Hi @SergeyKanzhelev , i dont know this API either. @cijothomas , can u tell me if we can or can not remove it?

Reply:
It is required, this is the context implementation for .NET Remoting, and we allow people to explicitly specify it based on the scenario/need.
Similar for the other XyzContextSlot<T>.

Reply:
should remoting one be only used when AsyncLocal is not available? Or there is a reason to have both?

Reply:
Need to have both and allow people to pick the non-default one.